src/Features/CSharp/Portable/Completion/KeywordRecommenders/IntKeywordRecommender.cs

```diff
@@ -37,6 +37,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                 context.IsFixedVariableDeclarationContext ||
                 context.IsParameterTypeContext ||
                 context.IsPossibleLambdaOrAnonymousMethodParameterTypeContext ||
+                context.IsLocalFunctionDeclarationContext ||
```

Comment:
i think this is proably better named as `IsLocalFunctionReturnTypeContext` (similar to `IsDelegateReturnTypeContext`).

Reply:
n/m.  i see this name already exists.